routers/web/explore/repo.go

```diff
@@ -142,10 +142,17 @@ func RenderRepoSearch(ctx *context.Context, opts *RepoSearchOptions) {
 		return
 	}
 
+	programLanguages, err := repo_model.GetPrimaryRepoLanguageList(ctx, 0, ctx.Doer)
```

Comment:
I'm not a native speaker but is this the usual word? Maybe `codingLanguages` or `programmingLanguages`?

Reply:
not a native speaker either ;)
but definitely agree that `programmingLanguages` does sound a lot better :)